.gitignore

```diff
@@ -32,6 +32,34 @@ GPATH
 **/examples/**/sdkconfig.old
 **/examples/**/build
 **/examples/**/managed_components
+common_examples/**/sdkconfig
+common_examples/**/sdkconfig.old
+common_examples/**/build
+common_examples/**/managed_components
+
+# Project files
+**/sdkconfig
+**/sdkconfig.old
+**/build
+**/managed_components
+
+# Project files
+**/sdkconfig
+**/sdkconfig.old
+**/build
+**/managed_components
+
+# Project files
+**/sdkconfig
+**/sdkconfig.old
+**/build
+**/managed_components
+
+# Project files
+**/sdkconfig
+**/sdkconfig.old
+**/build
+**/managed_components
```

Comment:
Copy-paste error?

Reply:
Change that was left out. Left this, removed duplicate for examples so that those project files are ignored everywhere.